<?xml version="1.0" encoding="utf-8" ?>
<launch>
    <arg name="topics" default="['/camera_r/depth/image_rect_raw', '/tf']"/>
    <arg name="path" default="/seerep/seerep-data/"/>
    <arg name="project_root_frame" default="map"/>
    <arg name="project_name" default="testproject"/>
    <arg name="max_viewing_distance" default="1.0"/>
    <node name="seerep_ros_examples_hdf5_node" pkg="seerep_ros_examples" type="seerep_ros_examples_hdf5_node" output="screen">
        <rosparam param="topics" subst_value="True">$(arg topics)</rosparam>
        <rosparam param="path" subst_value="True">$(arg path)</rosparam>
        <rosparam param="project_root_frame" subst_value="True">$(arg project_root_frame)</rosparam>
        <rosparam param="project_name" subst_value="True">$(arg project_name)</rosparam>
        <rosparam param="max_viewing_distance" subst_value="True">$(arg max_viewing_distance)</rosparam>
    </node>
</launch>
